### ISSUE TITLE
SearchItem -> NotIn bug

### ISSUE THREAD

**Issue by chsword** [fissoft/Fissoft.Core#1]

**Comment by chsword** [comment]
when use List<int> as a parameter

**Comment by chsword** [comment]
``` c#
 [TestMethod]
        public void NotInIntList()
        {
            var item = new SearchItem
            {
                Field = "Id",
                Method = SearchMethod.NotIn,
                Value = new List<int>{ 2, 3 }
            };
            var query = _table.AsQueryable();

            var actual = query.Where(item);
            Assert.AreEqual(1, actual.Count());
        }

```

**Comment by chsword** [comment]
fix @ArvinXie  v1.0.40
